### ISSUE TITLE
Versioning

### ISSUE THREAD

**Issue by phodina** [m-weigand/mw_pinenote_misc#11]
Hi @m-weigand,

could I ask you to provide versioning. I've created a package for the gnome-shell eink extension for PostmarketOS.

https://gitlab.com/postmarketOS/pmaports/-/merge_requests/4142/commits

**Comment by m-weigand** [comment]
Hi @phodina, I'm terribly sorry, I'm a little bit behind with my PineNote work items...one among them is to clean up this repo.
Anyway, the gnome extension got its own repository now, and I think this should provide enough versioning?

**Comment by phodina** [comment]
Hi @m-weigand ,
I've found it https://github.com/PNDeb/pinenote-gnome-extension/.

So I adapted the packaging for Alpine.

Thanks for making the extension. I used it with the simple `install.sh` script on previous versions of GNOME Shell.

Unfortunately with `v44.2` (maybe also some previous versions) the extension is disabled due to  being incompatible.

Will try to look into that and learn how to write extensions.
